phpstan.neon.dist

```diff
@@ -9,3 +9,9 @@ parameters:
     checkMissingCallableSignature: true
     checkUninitializedProperties: true
     checkTooWideReturnTypesInProtectedAndPublicMethods: true
+
+    ignoreErrors:
+        -
+            # CORE_VERSION constant may change in the future
+            message: '~Right side of && is always true~'
+            path: 'src/EditorconfigChecker/Utilities.php'
```

Comment:
This could theoretically ignore more than expected. I'd add `count: 1`.

Reply:
yep. you are right. here it is: https://github.com/editorconfig-checker/editorconfig-checker.php/pull/223

i always disliked this type of error messages. it is too generic. i will try to convince Ondřej Mirtes to include the actual expression in the error message so it is better readable and does not need to be filtered by file and count in all cases

Reply:
Yeah, but that will be messy for existing applications with ton of ignores.

Thanks, lets close this one.